libobs/obs-cocoa.m

```diff
@@ -93,6 +93,25 @@ static void log_processor_speed(void)
         blog(LOG_INFO, "CPU Speed: %lldMHz", freq / 1000000);
 }
 
+static void log_model_name(void)
+{
+    char *name = NULL;
+    size_t size;
+    int ret;
+
+    ret = sysctlbyname("hw.model", NULL, &size, NULL, 0);
+    if (ret != 0)
+        return;
+
+    name = malloc(size);
```

Comment:
Even if memory alignment isn't needed, I'd use `bmem.h`'s  `bmalloc` and `bfree` because it's used universally across OBS.
Maybe you could even use a fixed size buffer to avoid the heap allocation altogether (say of size 128 or something), since we know the string won't be infinitely long.

Also personally I'd rather do the declaration at the same place as the initialization, but I suppose that's just style preferences.

Reply:
Changed. Added a new commit to use the aligned managers in `log_processor_name()` as well.